docs/web_advanced.rst

```diff
@@ -393,6 +393,82 @@ and safe to use as the key.
 Otherwise, something based on your company name/url would be satisfactory (i.e.
 ``org.company.app``).
 
+
+.. _aiohttp-web-contextvars:
+
+
+ContextVars support
+-------------------
+
+Starting from Python 3.7 asyncio has :mod:`Context Variables <contextvars>` as a
+context-local storage (a generalization of thread-local concept that works with asyncio
+tasks also).
+
+
+*aiohttp* server supports it in the following way:
+
+* A server inherits a current task's context used by the server
+  creation. :func:`aiohttp.web.run_app()` runs a task for handling all underlying dirty
```

Comment:
> runs a task for handling all underlying dirty job

 Doesn't make sense to me. I'm not entirely sure what you're trying to say, but perhaps something like

```
A server inherits the current task's context used when creating it. 
:func:`aiohttp.web.run_app()` runs a task for handling all underlying jobs running 
the app, but alternatively :ref:`aiohttp-web-app-runners` can be used.
```